### PR TITLE
docs - prometheus

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
Rough draft of docs for setting up observability in Metabase with Prometheus. 

I scrounged up this doc just by reading through the product docs and testing locally, so I expect there to be a fair bit of inaccuracy and missing info.

@luizarakaki Can you give this a quick technical review?

- Everything correct?
- Missing any helpful info?

I'll follow up with @nllho for a review once you've given some feedback.

### CODE REVIEW

File: docs/installation-and-operation/observability-with-prometheus.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+---
+title: observability-with-prometheus
+---
+
+# Observability with Prometheus
+
+You can export [Prometheus](https://prometheus.io/) stats from your Metabase.
```
Comment (Onlinehead) [inline docs/installation-and-operation/observability-with-prometheus.md:7]: Formally, that is not a "prometheus stats", that is metrics in a Prometheus [format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md). Not only Prometheus supports it, same as not only Java JMX tools support JMX format.

File: docs/installation-and-operation/observability-with-prometheus.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+## Running Metabase and Prometheus locally
+
+To give you an idea of how Metabase and Prometheus would work in your production environment, we'll walk through how to set up Metabase and Prometheus locally. 
+
+### Start up Metabase with `MB_PROMETHEUS_SERVER_PORT`
```
Comment (paoliniluis) [inline docs/installation-and-operation/observability-with-prometheus.md:13]: should we specify that this works from 44+?

File: docs/installation-and-operation/observability-with-prometheus.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+Download the latest [Metabase JAR](https://www.metabase.com/start/oss/), and run Metabase using an environment variable to specify the Prometheus server port:
+
+```
+MB_PROMETHEUS_SERVER_PORT=9191 java -jar metabase.jar
+```
+
+The `MB_PROMETHEUS_SERVER_PORT=9191` specifies which port (`9191`) Metabase will use to send data to Prometheus. To clarify the ports that will be involved here:
+
+- Port `3000` is the port Metabase uses to serve the Metabase app. You can set another port with `MB_JETTY_PORT` (e.g., `MB_JETTY_PORT=3001`).
+- Port `9191` (or whichever port you specified with the `MB_PROMETHEUS_SERVER_PORT` environment variable) is the port Metabase uses to send metrics to Prometheus.
```
Comment (paoliniluis) [inline docs/installation-and-operation/observability-with-prometheus.md:24]: I would say here that 9191 is the port where Prometheus will scrape metrics from Metabase (apps don't push metrics to Prometheus, it's the other way around)

**Comment by paoliniluis** [comment]
https://github.com/metabase/metabase/issues/25745

**Review by dragonsahead — APPROVED** [review]

**Pull request merged**